repose-aggregator/services/service-client/api/src/main/java/org/openrepose/services/serviceclient/akka/AkkaServiceClientException.java

```diff
@@ -0,0 +1,19 @@
+package org.openrepose.services.serviceclient.akka;
+
+public class AkkaServiceClientException extends Exception {
```

Comment:
No. The only one used is the AkkaServiceClientException(String, Throwable). It was just the standard custom Exception template.

Do you want me to remove the other three?